src/ec/packages/ec-component-text-area/ecl-text-area.html.twig

```diff
@@ -110,7 +110,9 @@
     id="{{ _id }}"
   {% endif %}
     class="{{ _css_class }}"
+  {% if _name and _name is not empty %}
```

Comment:
_name is a local variable that we define at the top of the template, but its default is an empty string, so here you only need to check for "is not empty" ;)